src/components/Dashboard/TodoList.vue

```diff
@@ -1705,37 +1763,39 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
   /** Resumes a draft filing. */
   doResumeFiling (item: TodoItemIF): void {
     switch (item.name) {
+      case FilingTypes.AMALGAMATION: {
+        // navigate to Create UI to resume this Amalgamation
+        const amalgamationUrl = `${this.getCreateUrl}?id=${this.tempRegNumber}`
+        navigate(amalgamationUrl)
+        break
+      }
+
       case FilingTypes.ANNUAL_REPORT:
         // resume this Annual Report locally
         this.setARFilingYear(item.ARFilingYear)
         this.setArMinDate(item.arMinDate) // COOP only
         this.setArMaxDate(item.arMaxDate) // COOP only
         this.setNextARDate(item.nextArDate) // BEN/BC/CCC/ULC only
-        this.setCurrentFilingStatus(FilingStatus.DRAFT)
```

Comment:
I refactored this in Annual Report view, and it wasn't used for other filing types.